src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,37 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    public static final int CAPACITY = 12;
+    public static final int START_VALUE = -1;
+    public static final int MAX = 11;
```

Comment:
This constant is related to capacity of numbers array. Changing only one leads to an error (inconsistency).